### PR TITLE
fix(error): add error handing logics

### PR DESCRIPTION
**Pull request by Xunzhuo**

### CODE REVIEW

File: topology-generator/pkg/service/service.go [hunk 1]
```diff
@@ -28,6 +28,23 @@ func loadEndpoints(fileName string, endpoints *[]api.Endpoint) error {
 	return err
 }
 
+func loadEndpoints(fileName string, endpoints *[]api.Endpoint) error {
```
Comment (jmazzitelli) [inline topology-generator/pkg/service/service.go:31]: why is there a duplicate function here? `func loadEndpoints(fileName string, endpoints *[]api.Endpoint) error` is already defined on l ine 23. I think you want to modify that function, not add a duplicate.

**Review by jmazzitelli — CHANGES_REQUESTED** [review]

File: topology-generator/pkg/service/service.go (hunk 1 above)
Comment (Xunzhuo) [reply topology-generator/pkg/service/service.go]: Done, sorry I forgot to update codes

**Comment by Xunzhuo** [comment]
cc @leandroberetta

**Review by leandroberetta — APPROVED** [review]
Looks good to me, thanks for adding more error handling logic.

**Pull request merged**